docs/source/conf.py

```diff
@@ -27,11 +27,12 @@
 sys.path.insert(0, os.path.abspath(PATH_ROOT))
 sys.path.append(os.path.join(PATH_RAW_NB, ".actions"))
 
-_SHOULD_COPY_NOTEBOOKS = True
+_SHOULD_COPY_NOTEBOOKS = bool(int(os.getenv("SHOULD_COPY_PL_NOTEBOOKS", 1)))
```

Comment:
```suggestion
_SHOULD_COPY_NOTEBOOKS = bool(int(os.getenv("PL_SHOULD_COPY_NOTEBOOKS", 1)))
```
should it start with `PL` like all other env vars we have in lightning?

Reply:
can you update this info inside contributing.md?

Reply:
Updated!

Reply:
you forgot `SHOULD_COPY_PL_NOTEBOOKS` -> `PL_SHOULD_COPY_NOTEBOOKS`?

Reply:
I don't think it's required, as it is explicit to `make html`